### DISCUSSION TITLE
Compatibility with Blender 2.92 and Correct Versions?

### DISCUSSION THREAD

**Discussion by LordMcMutton** [bnpr/Malt#155] [Q&A] [answered]
Hello!

I'm currently stuck at Blender 2.92 due to my OS. Is the latest version of Malt compatible with it? Or is there a way to get a hold of a previous version that is?

I'm experiencing the same crashing issue reported in Issue #144, so I'd like to rule out incompatibilities before adding information to it.

**Comment by pragma37** [comment, marked as answer]
I think it should work, but you can always download the portable 2.93 version to be sure that's not the issue.
https://www.blender.org/download/

**Reply by LordMcMutton** [reply]
Looks like I can't open up that version on Windows 7, either, despite being able to install it.

If I'm experiencing the same result as the aforementioned issue, but for different reasons, should I add to that issue or create my own?

**Reply by pragma37** [reply]
I didn't know Blender 2.93 doesn't support Windows 7!
I just tested the latest Malt release in 2.92 and it works fine (Windows 10).
Feel free to open a new issue, but there's no plan to keep support for older Blender versions in the long run.

**Reply by LordMcMutton** [reply]
A shame, right? It's always something.

I'll go ahead and make a new issue- it'll help to figure out if it's Windows 7's fault or something that I can fix on my end.
